integration/spark/src/main/scala/org/apache/carbondata/geo/InPolygonUDF.scala

```diff
@@ -59,23 +58,3 @@ class InPolygonRangeListUDF extends ((String, String) => Boolean) with Serializa
     true // Carbon applies the filter. So, Spark do not have to apply filter.
   }
 }
```

Comment:
this filter is use for Spatial index， if remove how about that?

Reply:
1. it will still keep the udf and expression for Spatial index

2. In the old flow,  carbon converts spark expression to spark filter, and then converts spark filter to carbon expression.
So, after creating carbon expression, this filter will be removed from memory.
Now in the new flow, it will convert spark expression to carbon expression directly, and not need this filter.